<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknosiec</class>
 <widget class="QDialog" name="oknosiec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_IP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Port">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Serwer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Client</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_start_connect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop_disconnect">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ip1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>127</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ip2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ip3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ip4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_port">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tryb">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Online</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
